src/uct/ib/base/ib_verbs.h

```diff
@@ -71,6 +71,9 @@ static inline ucs_status_t uct_ib_query_device(struct ibv_context *ctx,
 #  define IBV_ACCESS_RELAXED_ORDERING               0
 #endif
 
+#if !HAVE_DECL_IBV_ACCESS_ON_DEMAND
+#  define IBV_ACCESS_ON_DEMAND                      0
+#endif
```

Comment:
why needed? can't we remove it from the code?

Reply:
Can you pls elaborate?
Avoiding compilation failure with arm clang on specific config

Reply:
AFAIR, we removed IBV_ACCESS_ON_DEMAND because it's supposed to be unused. Why do we need to return it?

Reply:
right..
so, maybe we need to expose from UCT that ODP is disabled if IBV_ACCESS_ON_DEMAND==0?
even if it's supported by FW